tools/shared.py

```diff
@@ -1168,7 +1168,7 @@ def reset(cls):
       # Load the JS defaults into python.
       settings = open(path_from_root('src', 'settings.js')).read().replace('//', '#')
       settings = re.sub(r'var ([\w\d]+)', r'attrs["\1"]', settings)
-      exec(settings, {'attrs': cls.attrs})
+      exec(settings, {'attrs': cls.attrs, 'TARGET_NOT_SUPPORTED': 0x7FFFFFFF})
```

Comment:
I guess this is too early for the numeric constant here to be `shared.Settings.TARGET_NOT_SUPPORTED`?

Please at least add a comment why this setting is special.